packages/ui/test/app/pages/Members.test.tsx

```diff
@@ -1,13 +1,13 @@
 import { cryptoWaitReady } from '@polkadot/util-crypto'
-import { render, screen } from '@testing-library/react'
+import { act, findAllByRole, fireEvent, render, screen } from '@testing-library/react'
```

Comment:
```suggestion
import { act, fireEvent, render, screen } from '@testing-library/react'
```

Seems like not used in the tests.